worker.ts

```
@@ -109,13 +112,26 @@ export class CustomTSWorker extends TypeScriptWorker {
 					containingFile,
 					options,
-					this, // <- 중요: host로 'this'를 전달하여 해당 class의 메서드를 사용하게 할수 있는듯
+					this,
 				)
-				if (standardResult.resolvedModule) {
+				if (
+					standardResult.resolvedModule === undefined &&
+					standardResult.failedLookupLocations.length > 0
+				) {
+					standardResult.failedLookupLocations.forEach((location) => {
+						location = location.replace("https:/", "https://")
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)